tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1459,6 +1492,14 @@ schedule(void)
     for (ilt = BIH_ILTFIRST(bih); ilt; ilt = ILT_NEXT(ilt))
       build_csed_list(ILT_ILIP(ilt));
 
+  /* process variables with parameter attribute */
+  if (!XBIT(49, 0x10)
```

Comment:
can this flag be added to xflag.n? What is its status by default?

Reply:
Why is it under an xflag to begin with? Wouldn't we want to have this debug information always available?

Reply:
I think I agree that we wouldn't want to bother putting this under an xflag. Once you do that, we have to make sure the driver sets it up as well, don't we? Now I have two pull requests.

Reply:
Sorry for the confusion here. The changes work with default option, No additional options are needed to make it work (Please notice NOT operator "!"). I just noticed some changes under this macro in semant.c and semant2.c and that suggests it to be some experimental option in absence of that which is by default the current PR would work. Please let me know if you want this to be removed.